judge/views/stats.py

```diff
@@ -62,6 +69,103 @@ def ac_rate(request):
     return JsonResponse(get_bar_chart(list(data)))
 
 
+def oj_data(request):
+    if request.method != 'POST' or not request.user.is_superuser or not request.user.is_staff:
```

Comment:
Please remove `or not request.user.is_staff`. Only super users can go to this site because of the heavy computation cost

Reply:
Fixed